tests/temp/tempFileTestInternalApi.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+require_once(dirname(__FILE__) . "/../../interface/globals.php");
+
+
+// Looking into how to use the api locally via php calls (4 methods are shown below)
```

Comment:
hi @sjpadgett ,
Have been playing around with the api a bit to figure out best way to support local calls from php. Gonna leave several posts in this script to cover each of 4 methods.